automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/Leonardo.scala

```diff
@@ -51,7 +51,8 @@ object Leonardo extends RestClient with LazyLogging {
                                     labels: LabelMap,
                                     jupyterExtensionUri: Option[String],
                                     jupyterUserScriptUri: Option[String],
-                                    stagingBucket: String) {
+                                    stagingBucket: String,
+                                    errors:List[ClusterError]) {
```

Comment:
This deserializes correctly in automation tests? For other nested structures we've specified `Map[String, String]`.

Reply:
Yep, it does deserialize correctly.